### PR TITLE
Feat: extract env0 host url to environment variable

### LINKED ISSUE #26: extract env0 host url to environment variable
Currently the API client calls to hardcoded "api-dev" domain, extract all usages to environment variable so the client can call to production domain

### PR DESCRIPTION
**Pull request by TomerHeber**
closes #26

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -3,33 +3,32 @@
...
+      "args": [
+        "--extensionDevelopmentPath=${workspaceFolder}",
+        "--extensionTestsPath=${workspaceFolder}/out/test/suite/index"
+      ],
+      "outFiles": [
+        "${workspaceFolder}/out/**/*.js",
+        "${workspaceFolder}/dist/**/*.js"
+      ],
+      "preLaunchTask": "tasks: watch-tests",
+      "env": {
```
Comment (TomerHeber) [inline .vscode/launch.json:29]: My IDE autoformats the json.
This is the new lines.

File: src/extension.ts [hunk 2]
```diff
@@ -16,7 +16,8 @@ import {
 } from "./env0-environments-provider";
 import { getEnvironmentsForBranch } from "./get-environments";
 
-export const ENV0_BASE_URL = "api-dev.dev.env0.com";
+export const ENV0_BASE_URL = process.env.ENV0_BASE_URL || "api.env0.com";
```
Comment (TomerHeber) [inline src/extension.ts:19]: If the env variable is not set use the production URL.
Tested locally with the dev api.

**Review by liranfarage89 — COMMENTED** [review]
we still have a `dev` leftover here: 
https://github.com/env0/vscode-env0/blob/feat-extract-host-url-%2326/src/notification-messages.ts#L53

@TomerHeber

File: src/actions.ts [hunk 3]
```diff
@@ -24,7 +24,7 @@ export const abortEnvironmentDeploy = (env: Environment) => {
     return;
   }
 
-  const abortDeploymentUrl = `https://${ENV0_BASE_URL}/environments/deployments/${id}/abort`;
+  const abortDeploymentUrl = `https://${ENV0_API_URL}/environments/deployments/${id}/abort`;
```
Comment (TomerHeber) [inline src/actions.ts:27]: since there are two "bases" one will be called API the other WEB...

**Review by liranfarage89 — APPROVED** [review]
Nice

**Pull request merged**